docs/app/pages/components/components-sections/wizard/marquee-wizard/marquee-wizard.component.html

```diff
@@ -258,10 +259,6 @@ <h1 class="m-t-nil">Sixth step title</h1>
     <tr uxd-api-property name="finishAriaLabel" type="string">
         Defines the text for the aria label on the 'Finish' button.
     </tr>
-    <tr uxd-api-property name="nextDisabled" type="boolean">
```

Comment:
Is this supposed to have been removed? As this is still an Input on the component?

Reply:
I will add in now